letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
oh dear, so I did it wrong in the first place. I wanted to avoid having to migrate existing installations, but this won't be necessary indeed, and the case I've in mind doesn't work with either. Sorry for the noise then.

Just make clear then that loading the module server wide is only necessary if the dbfilter doesn't restrict to exactly one database.